conf.py

```diff
@@ -2,3 +2,7 @@
 # https://documenteer.lsst.io/technotes/
 
 from documenteer.conf.technote import *  # noqa F401 F403
+
```

Comment:
I think you'll need to add this to use the css file in _static

`html_static_path = ['_static']
`